newIDE/app/src/Utils/UseOpenInitialDialog.js

```diff
@@ -31,6 +35,16 @@ export const useOpenInitialDialog = ({
           openSubscriptionDialog({ reason: 'Landing dialog at opening' });
           removeRouteArguments(['initial-dialog']);
           break;
+        case 'signup':
+          const timeoutId = setTimeout(() => {
```

Comment:
is the timeout to let the authentication some time to initialize?
I'd add a comment explaining why this is important

Reply:
Indeed, I'll add the comment